client/electron/electron-builder.json

```diff
@@ -15,21 +16,35 @@
     "output/client/electron",
     "!output/client/electron/build"
   ],
+
+  "deb": {
+    "depends": [
+      "gconf2", "gconf-service", "libnotify4", "libappindicator1", "libxtst6", "libnss3",
```

Comment:
These are electron required dependencies: https://www.electron.build/app-builder-lib.interface.deboptions#depends

Reply:
Can you add them as comments to the file? Ugh, it's JSON...

Reply:
Right, JSON is bad. How about adding the field `"_depends_comment": "there are electron required deps"`?